redbot/core/utils/chat_formatting.py

```diff
@@ -283,6 +284,103 @@ def pagify(
             yield in_text
 
 
+def rows_to_embeds(
+    rows: List[str],
+    base_embed: discord.Embed,
+    footer_static: str = None,
+    embed_max_fields: int = 2,
+    field_max_rows: int = 24,
+    greedy_fill: bool = False,
+    always_page_number: bool = False,
+) -> List[discord.Embed]:
+    """
+    Parameters
+    ----------
+    rows : `list` of `str`
+        The strings that form a row in the embed
+    base_embed : `discord.Embed`
+        The embed to use as a base for the list of embeds
+    embed_max_fields : `int`, optional
```

Comment:
The optional typing here should be Optiona[type] just to follow the existing format, this also applies to the other instances in the docstring